tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "@mskelton/tsconfig",
   "compilerOptions": {
-    "rootDir": "src",
-    "noEmit": true
+    "allowSyntheticDefaultImports": true,
```

Comment:
I'd prefer not to enable this. Instead can you do `import * as naturalCompare`?

Reply:
I actually don't fully understand what `allowSyntheticDefaultImports` does, but disabling it and using `import * as naturalCompare` doesn't seem to solve the issue

![Uploading Screenshot 2023-03-08 at 07.44.27.png…]()

Reply:
Sadness